### PR TITLE
[Multi-asic]: Namespace support for LLDP and Sensor tables

### PR DESCRIPTION
**Pull request by SuvarnaMeenakshi**
Signed-off-by: SuvarnaMeenakshi <sumeenak@microsoft.com>

**- What I did**
This PR targets changes in lldp/sensor related MIB implementation for multi-asic platform.
LLDP Mgmt interface related information will be obtained from global db.
Update implementation of the following MIBS to get data from all
databases:
    LLDPRemManAddrTable
    PhysicalTableMIB
    PhysicalSensorTableMIB
    QBridgeMIBObjects
**- How I did it**
Modified implementation of above MIB classes to obtain data from all namespace database DBs.

**- How to verify it**
Verified in multi-asic platform.
Note: Requires code/test mock tables from https://github.com/Azure/sonic-snmpagent/pull/126 

**- Description for the changelog**

### CODE REVIEW

File: src/sonic_ax_impl/mibs/ieee802_1ab.py [hunk 1]
```diff
@@ -421,12 +426,11 @@ def update_data(self):
         Subclass update data routine. Updates available LLDP counters.
         """
         # establish connection to application database.
-        self.db_conn.connect(mibs.APPL_DB)
 
         self.if_range = []
         self.lldp_counters = {}
         for if_oid, if_name in self.oid_name_map.items():
-            lldp_kvs = self.db_conn.get_all(mibs.APPL_DB, mibs.lldp_entry_table(if_name))
+            lldp_kvs = Namespace.get_all_dbs(self.db_conn, mibs.APPL_DB, mibs.lldp_entry_table(if_name))
```
Comment (qiluo-msft) [inline src/sonic_ax_impl/mibs/ieee802_1ab.py:433]: >get_all_dbs [](start = 33, length = 11)

dbs_get_all

File: src/sonic_ax_impl/mibs/__init__.py [hunk 2]
```diff
@@ -419,7 +419,11 @@ def update_data(self):
         self.oid_list = []
         self.oid_map = {}
 
+<<<<<<< HEAD
```
Comment (qiluo-msft) [inline src/sonic_ax_impl/mibs/__init__.py:422]: Please resolve conflicts

File: src/sonic_ax_impl/mibs/__init__.py [hunk 3]
```diff
@@ -571,3 +595,22 @@ def init_namespace_sync_d_queue_tables(dbs):
             port_queue_list_map.update(port_queue_list_map_ns)
 
         return port_queues_map, queue_stat_map, port_queue_list_map
+
+    @staticmethod
+    def get_bridge_port_map_from_namespace_dbs(dbs, db_name):
```
Comment (qiluo-msft) [inline src/sonic_ax_impl/mibs/__init__.py:600]: >get_bridge_port_map_from_namespace_dbs [](start = 8, length = 38)

dbs_get_bridge_port_map

File: src/sonic_ax_impl/mibs/__init__.py [hunk 4]
```diff
@@ -571,3 +595,22 @@ def init_namespace_sync_d_queue_tables(dbs):
...
+        get_bridge_port_map from all namespace DBs
+        """
+        if_br_oid_map = {}
+        for db_conn in Namespace.get_non_host_dbs(dbs):
+            if_br_oid_map_ns = port_util.get_bridge_port_map(db_conn)
+            if_br_oid_map.update(if_br_oid_map_ns)
+        return if_br_oid_map
+
+    @staticmethod
+    def get_vlan_id_from_bvid_from_namespace_dbs(dbs, bvid):
```
Comment (qiluo-msft) [inline src/sonic_ax_impl/mibs/__init__.py:611]: >get_vlan_id_from_bvid_from_namespace_dbs [](start = 8, length = 40)

dbs_get_vlan_id_from_bvid

File: src/sonic_ax_impl/mibs/ieee802_1ab.py (hunk 1 above)
Comment (SuvarnaMeenakshi) [reply src/sonic_ax_impl/mibs/ieee802_1ab.py]: Fixed.

File: src/sonic_ax_impl/mibs/__init__.py (hunk 2 above)
Comment (SuvarnaMeenakshi) [reply src/sonic_ax_impl/mibs/__init__.py]: Fixed.

File: src/sonic_ax_impl/mibs/__init__.py (hunk 3 above)
Comment (SuvarnaMeenakshi) [reply src/sonic_ax_impl/mibs/__init__.py]: Fixed.

File: src/sonic_ax_impl/mibs/__init__.py (hunk 4 above)
Comment (SuvarnaMeenakshi) [reply src/sonic_ax_impl/mibs/__init__.py]: Fixed.

**Pull request merged**